### DISCUSSION TITLE
WSL Docker Magento Installation Problem

### DISCUSSION THREAD

**Discussion by ronakslamba** [markshust/docker-magento#1154] [Q&A]
Trying to install docker magento in Windows and ran into this after running the curl command. Any ideas?
![magento install error1](https://github.com/markshust/docker-magento/assets/157072747/da770b0c-64a4-4d14-8763-edb6e29dad01)

**Comment by YevhenZvieriev** [comment]
Hi, @ronakslamba 

How much  RAM do you have? There must be at least 6GB.
